openy_gated_content.install

```diff
@@ -158,3 +158,16 @@ function openy_gated_content_update_8007() {
     }
   }
 }
+
+/**
+ * Update virtual_ymca_editor user role.
+ */
+function openy_gated_content_update_8008() {
+  $config_dir = drupal_get_path('module', 'openy_gc_storage');
```

Comment:
Wrong module name, should be `openy_gated_content`

Reply:
I've did this fix